editor/script_create_dialog.h

```diff
@@ -124,6 +133,7 @@ class ScriptCreateDialog : public ConfirmationDialog {
 public:
 	void config(const String &p_base_name, const String &p_base_path, bool p_built_in_enabled = true, bool p_load_enabled = true);
 	void set_inheritance_base_type(const String &p_base);
+	void set_languages_list(const bool p_only_attachable);
```

Comment:
```suggestion
	void set_languages_list(bool p_only_attachable);
```
We don't use `const` with primitive method parameters.